src/goto-analyzer/unreachable_instructions.cpp

```diff
@@ -152,8 +152,8 @@ static void add_to_json(
 
     // print info for file actually with full path
     const source_locationt &l=it->second->source_location;
-    json_objectt i_entry(
-      {{"sourceLocation", json(l)}, {"statement", json_stringt(s)}});
+    json_objectt i_entry{{"sourceLocation", json(l)},
```

Comment:
Surprising formatting -- break after first `{`?

Reply:
If I insert a new line, then clang format will just remove it again. I agree that left alignment of the enclosed pairs would be preferable. However I prefer the sub optimal formatting over the extra comments to switch off the formatting in this case.